lib/hal/rpihal.ex

```diff
@@ -2,8 +2,9 @@ defmodule Inky.RpiHAL do
   @default_io_mod Inky.RpiIO
 
   @moduledoc """
-  An `Inky.HAL` implementation responsible for sending commands to the Inky screen. It delegates
-  to whatever IO module its user provides at init, but defaults to #{inspect(@default_io_mod)}
+  An `Inky.HAL` implementation responsible for sending commands to the Inky
+  screen. It delegates to whatever IO module its user provides at init, but
+  defaults to #{inspect(@default_io_mod)}
```

Comment:
Dark magic, you can inject module attributes into docs. I guess its fine though it is a bit weird for in-code readability :)

Reply:
⭐️ Teachable moment ⭐️ 

(module)docs are just strings that are set as module attributes and later interpreted in special ways. This is just regular string interpolation ^^

Reply:
Oh, sure. I understood the string interpolation and figured it was a normal string. I hadn't considered it had access to the module context :)